src/api/shop.js

```diff
@@ -0,0 +1,16 @@
+const {
```

Comment:
What exactly does this file do?

Reply:
The file creates a function that sends a JSON object. I was trying out a tutorial that used this so added it in as trial. But I don't actually make use of this function anywhere so the file should be removed